### PR TITLE
Adding Ember Front End

### PR DESCRIPTION
**Pull request by niteshkumarniranjan**
Fixes #

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Created New Front End With Ember.js

### CODE REVIEW

**Comment by niteshkumarniranjan** [comment]
Fixed #388

**Comment by djmgit** [comment]
@niteshkumarniranjan I guess the test failures have got nothing to do with your changes. There are caused due to some other reason. In your last commit you have deleted all the files. Are you working on them?

**Comment by niteshkumarniranjan** [comment]
No I just Reuploaded @djmgit

**Comment by niteshkumarniranjan** [comment]
I thought this will fix the failure of Travis.

**Comment by djmgit** [comment]
@niteshkumarniranjan the failures are not related to this PR.
@vaibhavsingh97 @ParthS007 @S2606 what are your views on this?

**Comment by niteshkumarniranjan** [comment]
@S2606 What you mean.

**Comment by S2606** [comment]
@niteshkumarniranjan What I mean is what is this PR trying to solve??

**Comment by niteshkumarniranjan** [comment]
Sorry! @S2606 I just mistakenly deleted those files while I tried to resolve travis.

**Comment by niteshkumarniranjan** [comment]
@S2606 Re Added.

**Comment by S2606** [comment]
@niteshkumarniranjan  as per the deplyment http://badgeyay-frontend.surge.sh/ nothing comes under add default image

**Comment by niteshkumarniranjan** [comment]
The API is not ready so, I had left it for the time when API will release.

**Comment by djmgit** [comment]
@S2606 This is a dummy UI done as a part of GCI-2017, @niteshkumarniranjan  has successfully completed the task, this is his PR. However the build is failing. What I think is the failures are caused by something else, not this PR. What do you feel?

**Comment by S2606** [comment]
@djmgit regarding travis it can be fixed in different PR

**Comment by vaibhavsingh97** [comment]
@djmgit What I think that, we should not merge this PR right now as some UI tweaks needed to be done. After that we can merge this 😄

**Comment by niteshkumarniranjan** [comment]
@vaibhavsingh97 What Tweaks do you need???

File: niteshkumarniranjan/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+addons:
+  chrome: stable
+
+cache:
+  directories:
+    - $HOME/.npm
+
+env:
+  global:
+    # See https://git.io/vdao3 for details.
```
Comment (harshit98) [inline niteshkumarniranjan/.travis.yml]: Why do we need another `travis.yml` file?

File: niteshkumarniranjan/.watchmanconfig [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "ignore_dirs": ["tmp", "dist"]
+}
```
Comment (harshit98) [inline niteshkumarniranjan/.watchmanconfig]: Any reason to add this file?

**Comment by djmgit** [comment]
@mariobehling @vaibhavsingh97 @harshit98 creating a ember font end for badgeyay is a GCI task, it was required to send the code in a folder named after the Github id of the participant. There are still 6 instances remaining. So should we not wait for the remaining instances to completed before finalising this one? I mean, that way we will get different options and we can move forward with the one which looks most suitable. What are your views?

**Comment by harshit98** [comment]
@djmgit I agree with you. We should wait for the remaining instances for more frontend design options. 💯

**Comment by vaibhavsingh97** [comment]
@niteshkumarniranjan Can you please conflicts then we will merge

**Comment by vaibhavsingh97** [comment]
Also, Add one Readme.md file explaining how to deploy it locally with the live demo link

**Comment by niteshkumarniranjan** [comment]
@vaibhavsingh97 Things I have to do:
1. Clear Conflicts
2. Add a Readme

Anything else....

**Comment by vaibhavsingh97** [comment]
Good Job @niteshkumarniranjan  🎉 ✨

**Pull request merged**